exporter/metric/metric_test.go

```diff
@@ -20,15 +20,20 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/googleinterns/cloud-operations-api-mock/cloudmock"
+	"github.com/stretchr/testify/assert"
+
```

Comment:
nit: these imports should be together with the one below (i.e. no empty line)

Reply:
Surprising that the linter didn't pick this up. Maybe we need to add some more CI steps